calcom/cal.com
# Fix disabled locations when rescheduling

## What does this PR do?

When rescheduling an event the radio buttons for choosing a location were disabled without anything selected. With this PR the disabled radio buttons are not shown anymore but the location of the previous booking which will also be the location of the rescheduled booking is shown instead. 

Also fixes: The input field for the phone number is not shown when Attendee Phone Number is selected as the first location. Only appears after choosing Organizer Phone Number and then again Attendee Phone Number. It also can be booked like that (so the booking location is then 'phone') (https://github.com/calcom/cal.com/issues/4981#issuecomment-1277623403)

Fixes #4981

 Loom Video: https://www.loom.com/share/53fde81383a84f8aa973fefd9668f505

**Environment**: Staging(main branch) 

## Type of change


## How should this be tested?

1. Reschedule a booking of an event type with several locations
2. See that it shows the location of the previous booking and not the radio buttons

1. Book an event with several locations and Attendee Phone Number should be the first location 
2. See that phone input is shown when Attendee Phone Number is default selected as the first location
